### PR TITLE
Update to and for PureScript 0.11

### PR DESCRIPTION
**Pull request by garyb**

### CODE REVIEW

File: src/GulpPurescript/Options.purs [hunk 1]
```diff
@@ -141,121 +142,103 @@ newtype PathArray = PathArray (Array String)
...
-  read val = read val >>= (\a -> case a of
-                                      "markdown" -> Right Markdown
-                                      "etags" -> Right ETags
-                                      "ctags" -> Right CTags
-                                      b -> Left $ TypeMismatch "Format" b)
+readPsc :: Foreign -> F Psc
+readPsc obj = do
+  src <- readSources =<< readProp srcKey obj
+  output <- readPropNU readString outputKey obj
+  noTco <- readPropNU readBoolean noTcoKey obj
```
Comment (paf31) [inline src/GulpPurescript/Options.purs]: I think some of these options are no longer available.

Comment (garyb) [inline src/GulpPurescript/Options.purs]: Ah yeah! I didn't revise the options at all... will do now.

**Comment by garyb** [comment]
Ok, updated the compiler options and renamed the tasks to match the commands now, so it's `plugin.compile` rather than `plugin.psc`, etc.

**Comment by paf31** [comment]
👍 Looks good to me!

**Comment by garyb** [comment]
@ethul can you review/merge/release when you get a chance please? No particular hurry on my behalf, but just thought I'd ping you as a reminder, since you're the official maintainer for this I think. :)

**Comment by ethul** [comment]
Will do! Thanks for the update on this. I can do the release this weekend.

**Comment by ethul** [comment]
Looks great! Thanks again

**Pull request merged**